src/middleware/tlsAuthentication.js

```diff
@@ -2,7 +2,7 @@
 
 import logger from 'winston'
 import pem from 'pem'
-import {rootCas as rootCAs} from 'ssl-root-cas/latest'
+import {rootCas as rootCAs} from 'ssl-root-cas'
```

Comment:
Does this need updating ?

Reply:
I was getting error about not finding latest

Reply:
We need to use latest here for security reasons. What error are you getting?

https://www.npmjs.com/package/ssl-root-cas#using-the-latest-certificates

Reply:
![image](https://user-images.githubusercontent.com/48720958/217755871-a6fbda48-d852-480e-b1e7-4834e6c644c3.png)

I will bring it back anyway, ad it is saying ('not a big deal')